src/main/java/frc/robot/subsystems/loader/LoaderPID.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.loader;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpiutil.math.MathUtil;
+import frc.robot.Robot;
+
+import static frc.robot.Robot.robotConstants;
+
+public class LoaderPID extends CommandBase {
+  private PIDController pidController;
+  private DoubleSupplier desiredVelocity;
+  private BooleanSupplier isFinished;
+  private boolean isTuning;
+  private SimpleMotorFeedforward feedForward;
+
+  /**
+   * This does PID on the loader based on the desired velocity
+   */
+  public LoaderPID(DoubleSupplier desiredVelocity, BooleanSupplier isFinished, boolean isTuning) {
```

Comment:
Add more constructors that receive a double setpoint and that have isTuning defaulted to 0